apps/fluent-tester/src/TestComponents/Avatar/CustomizedAvatar.tsx

```diff
@@ -65,6 +65,9 @@ export const CustomizeUsage: React.FunctionComponent = () => {
     viewBox: '0 0 500 500',
   };
 
+  const useJavaHashCode = ['ios', 'macos', 'android'].includes(Platform.OS);
+  const hashedColor = AvatarColors[getJavaHashCode(name) % (AvatarColors.length - 3)];
+
```

Comment:
Instead of doing this in the test, could we just make this the default in Avatar for these platforms?

Reply:
(Sorry, I couldn't comment before this merged)

Reply:
Yeah, otherwise users of the Avatar in these platforms will have to do the same thing to get the expected behavior.